src/client.js

```diff
@@ -4800,6 +4804,29 @@ MatrixClient.prototype.startClient = async function(opts) {
     this._clientOpts = opts;
     this._syncApi = new SyncApi(this, opts);
     this._syncApi.sync();
+
+    if (opts.clientWellKnownPollPeriod !== undefined) {
+        this._clientWellKnownInterval =
```

Comment:
we should probably call this a `timerId` of sorts, as it sounds like a number (even though timer IDs are numbers...)